### DISCUSSION TITLE
Is there any way configure multiple SUFeedURLs?

### DISCUSSION THREAD

**Discussion by Fidetro** [sparkle-project/Sparkle#2543] [Q&A]
Some countries prohibit the use of country-specific servers, is there a way to configure multiple SUFeedURLs or is there another solution?

**Comment by zorgiepoo** [comment]
For multiple feeds, there is a section in the documentation for [configuring the feed url programmatically](https://sparkle-project.org/documentation/publishing/#setting-the-feed-programmatically).
